blueocean-dashboard/src/main/js/jenkins-js-extension.yaml

```diff
@@ -2,8 +2,8 @@
 # NB: "component" currently maps to modules, not "symbols" so make sure to "export default"
 # WARNING: If you change this you'll have to change io.jenkins.blueocean.jsextensions.JenkinsJSExtensionsTest as well :(
 extensions:
-  - component: AdminNavLink
-    extensionPoint: jenkins.logo.top
```

Comment:
does this extension point still exist?

Reply:
it is used in our internal project, so good you left the extensionpoint.

Reply:
@scherler good point

@cliffmeyers reminder: we should not remove/move extension points (add new ones) - think of it as a 18 karat gold rule. Not 24 karat as there may be exceptions, but 18 karat.